### PR TITLE
[DOCS-9857] Minor copy edits for the OC CasC files

### PR DESCRIPTION
**Pull request by kellie-freeman**
Minor copy edits for consistency with the recently revised CloudBees CI add-on for Amazon EKS blueprints.

### CODE REVIEW

File: bp02/jcasc/main.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ unclassified:
   bundleStorageService:
     activated: true
     activeBundle:
-      name: "casc-mm-store"
+      name: "casc-mc-store"
```
Comment (kellie-freeman) [inline bp02/jcasc/main.yaml:8]: @carlosrodlop - I updated this to `casc-mc-store`, but please revert this change if it will cause problems.

File: README.md [hunk 2]
```diff
@@ -2,8 +2,8 @@
 
 [![gitleaks badge](https://img.shields.io/badge/protected%20by-gitleaks-blue)](https://github.com/zricethezav/gitleaks#pre-commit) [![gitsecrets](https://img.shields.io/badge/protected%20by-gitsecrets-blue)](https://github.com/awslabs/git-secrets)
 
-Repository for [Configuration as Code for the operations center](https://docs.cloudbees.com/docs/cloudbees-ci/latest/casc-oc/).
+This repository is for [Configuration as Code (CasC) for the operations center](https://docs.cloudbees.com/docs/cloudbees-ci/latest/casc-oc/).
 
-Plugins and plugin catalogs curated with [casc-plugin-dependency-calculation](https://github.com/kyounger/casc-plugin-dependency-calculation).
+Plugins are curated with the [CloudBees CasC Plugin Catalog and Transitive Dependencies Calculator](https://github.com/kyounger/casc-plugin-dependency-calculation).
```
Comment (kellie-freeman) [inline README.md:7]: @carlosrodlop - I removed the reference to plugin catalogs here, since I don't think they're supported for OC CasC bundles. If this is incorrect, please revert this change.

**Review by carlosrodlop — APPROVED** [review]
